src/devices/Nmea0183/TalkerSentence.cs

```diff
@@ -220,7 +223,7 @@ public TalkerSentence(NmeaSentence sentence)
             }
 
             int firstComma = sentence.IndexOf(',', 1);
-            if (firstComma == -1)
+            if (firstComma <= 3)
```

Comment:
why 3?

Reply:
See the lines below. The sentence Id starts at message character 3 and cannot have a negative length.